tests/integration_test/integration_test.sh

```diff
@@ -287,11 +287,14 @@ docker run -i --rm \
     -e INPUT_PATH=/working \
     -e OUT_PATH=/working \
     -e ENV=${env} \
+    -e LIRA_VERSION="int_test_lira" \
```

Comment:
This should be the `$lira_version` variable that gets passed in, right?

Reply:
Yes, I assumed we don't care about this on-the-fly Lira in the integration test, but using `$lira_version` here is the should-be way. Will make a change.